lib/assessment.rb

```diff
@@ -5,7 +5,12 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.reduce do |item, memo|
+      if memo.to_s.length > item.to_s.length
+        longest = memo
+      else longest = item
+      end
+    end
```

Comment:
Otherwise, this is correct. However, it could have been written much more shortly. Let's do some refactoring - we can replace the if..else with a ternary operation and condense the whole thing to a single line.
`enum.reduce { |memo, item| memo.length > item.length? memo : item }`